src/main/java/org/jenkinsci/plugins/ghprb/GhprbRepository.java

```diff
@@ -144,16 +144,12 @@ public void createCommitStatus(AbstractBuild<?, ?> build, String sha1, GHCommitS
             } else {
                 ghRepository.createCommitStatus(sha1, state, url, message);
             }
-        } catch (FileNotFoundException ex) {
-            newMessage = "FileNotFoundException means that the credentials Jenkins is using is probably wrong. Or that something is really wrong with github.";
-            if (stream != null) {
-                stream.println(newMessage);
-                ex.printStackTrace(stream);
+        } catch (IOException ex) {
+            if (ex instanceof FileNotFoundException) {
+              newMessage = "FileNotFoundException means that the credentials Jenkins is using is probably wrong. Or the user account does not have write access to the repo.";
```

Comment:
Looking at the unified and split diff of the change, it does look a bit odd.

Basically I removed the FNF exception (since it's also an IOException).  Inside the IOException catch I check for the FNF and set the message accordingly.  Then I process the rest of the IOException as normal (the fallback posting was previously only done for non-FNF IOExceptions)